poe-api/src/poe.py

```diff
@@ -147,6 +147,16 @@ def get_bot_names(self):
       bot_names[bot_nickname] = bot_obj["displayName"]
     return bot_names
   
+  def get_remaining_messages(self, chatbot):
+    url = f'https://poe.com/_next/data/{self.next_data["buildId"]}/{chatbot}.json'
+    limited_chatbots = ["GPT-4", "Claude+"]
```

Comment:
There's no need to hardcode the bot names, that'll just cause the function to break whenever Poe changes their bots.